metron-interface/metron-alerts/package-lock.json

```diff
@@ -1,6427 +0,0 @@
-{
```

Comment:
I suspect the build failure is due to a lack of a license on this file.  Since JSON cannot take comments, you'll need to adjust the top level pom to add an exclusion for `**/package-lock.json` [here](https://github.com/apache/metron/blob/master/pom.xml#L304)  Just make a quick comment above it and note that this is yarn collateral.

Reply:
I think that was the problem!
Thanks. @cestella